optaplanner-wb-webapp/src/main/resources/org/optaplanner/workbench/client/resources/i18n/AppConstants.properties

```diff
@@ -17,7 +16,6 @@ HomeProducer.ProjectAuthoring=Project Authoring
 HomeProducer.ArtifactRepository=Artifact Repository
 HomeProducer.Administration=Administration
 HomeProducer.DeployYourArtifacts=Deploy your Artifacts
-
```

Comment:
The code formatter might not have been tested propertly on non-java files.
I suggest we apply it first on java files, jsp files - that's the far majority), but do other files types on a case by case basis.